hack/render-manifests.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+	"text/template"
+
+	"github.com/pkg/errors"
+)
+
+func exitWithError(err error, cause string, args ...interface{}) {
+	fmt.Fprintf(os.Stderr, "render-manifests.go: error: %v\n", errors.Wrapf(err, cause, args))
+	os.Exit(1)
+}
+
+func main() {
+	type Inventory struct {
+		Namespace           string
+		NMStateHandlerImage string
+		ImagePullPolicy     string
+	}
+
+	inventory := Inventory{
+		Namespace:           os.Args[1],
+		NMStateHandlerImage: os.Args[2],
+		ImagePullPolicy:     os.Args[3],
+	}
+	inputDir := os.Args[4]
+	outputDir := os.Args[5]
```

Comment:
Please count passed parameters first so we can fail with a user-friendly error if they don't provide enough arguments.